docker-compose.yml

```diff
@@ -20,8 +24,34 @@ services:
     networks:
       - backend
 
+  mysql:
+    image: mysql
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+      - MYSQL_ROOT_PASSWORD=natlas-dev-password-do-not-use
+      - MYSQL_DATABASE=natlas
+    networks:
+      - backend
+
+  zipkin-server:
+    image: openzipkin/zipkin-slim
+    ports:
+      - 9411:9411
+    networks:
+      - backend
+
+  opencensus-agent:
+    image: omnition/opencensus-agent
```

Comment:
We've been using explicit versions elsewhere. Is there any reason why we shouldn't use them here too?

Reply:
The only reason I could think of is that we don't have an explicit version dependency on any of these, whereas with node/python/ubuntu we have dependencies that rely on those versions. That said, explicit versions is a good idea just for reproducibility so I'll add them in.